build.gradle

```diff
@@ -8,7 +8,7 @@ idea.project.languageLevel = '1.8'
 
 subprojects {
 
-    version='0.8.4-SNAPSHOT'
+    version='0.9.1-SNAPSHOT'
```

Comment:
I would actually like to have the 0.9.0 checked in as a separate commit. This way you can tag this version and you can later check it out and build it with the correct version number.